code/src/components/HappyThought.js

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import moment from "moment";
+import "./happyThought.css";
+
+export const HappyThought = props => {
+    const {message, hearts, createdAt, _id} = props.thought
+    
+    const handleClick = () => {
+        fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts/${_id}/like`, {
+            method: "POST",
+            body:"",
```

Comment:
Maks told me that the _body:""_  wasn’t necessary when handling clicks on the hearts. 😇 But he can maybe explain it better...

Reply:
Exactly ;) It's because all necessary informations are being passed inside URL in line 9